iree_tests/configs/config_gpu_rocm_models.json

```diff
@@ -12,7 +12,10 @@
     "skip_compile_tests": [
       "sdxl-scheduled-unet-3-tank",
       "sdxl-vae-decode-tank",
-      "sdxl-prompt-encoder-tank"
+      "sdxl-prompt-encoder-tank",
+      "sd3-mmdit",
+      "sd3-vae-decode",
+      "sd3-prompt-encoder"
```

Comment:
If this is going to exclude all these special models they should be moved to a different subdirectory.

The `config_*_models` configs were intended to test a set of models that were imported into the test suite in a uniform way and each of these models is special in some way.